src/containers/ListPageLayout/ListPageLayout.js

```diff
@@ -0,0 +1,104 @@
+/*
+Copyright 2020 The Tekton Authors
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+/* istanbul ignore file */
+import React from 'react';
+import { connect } from 'react-redux';
+import { generatePath } from 'react-router-dom';
+import { ALL_NAMESPACES, paths } from '@tektoncd/dashboard-utils';
+
+import { selectNamespace as selectNamespaceAction } from '../../actions/namespaces';
+import { getSelectedNamespace, getTenantNamespace } from '../../reducers';
+import { LabelFilter, NamespacesDropdown } from '..';
+
+import './ListPageLayout.scss';
+
+const ListPageLayout = ({
+  children,
+  filters,
+  hideNamespacesDropdown,
+  history,
+  location,
+  match,
+  namespace,
+  selectNamespace,
+  tenantNamespace,
+  title
+}) => {
+  function setPath(path) {
+    history.push(path + location.search);
```

Comment:
up to you on the whole using template strings or not

Reply:
yeah we use them in most places so I'll update these for consistency 👍